javascript/javascript1/week1/Goodboy-Oldboy.js

```diff
@@ -0,0 +1,14 @@
+//Dog age calculator
+const dogYearOfBirth = 2017;
```

Comment:
I believe this variable should be changed to this...since it's much clearer what you are trying to calculate.
Same goes for the next one. What do you think?
```suggestion
const humanYearOfBirth = 2017;
```

Reply:
Yes, Made the changes